### ISSUE TITLE
BUG port missing

### ISSUE THREAD

**Issue by alphonse92** [jcrodriguez-dis/vpl-xmlrpc-jail#34]
The latest version does not set the jail por to the jail url monitor:

<img width="805" alt="Screen Shot 2019-11-19 at 12 23 04 PM" src="https://user-images.githubusercontent.com/18203585/69170095-7cfdd100-0ac7-11ea-82b8-2b6819965aa8.png">

<img width="751" alt="Screen Shot 2019-11-19 at 12 24 32 PM" src="https://user-images.githubusercontent.com/18203585/69170128-90a93780-0ac7-11ea-933f-c4aa76f87845.png">

**Comment by alphonse92** [comment]
i tried with another port, but i cant connect to it via WS to the monitor. But the same

**Comment by jcrodriguez-dis** [comment]
I have not been able to reproduce the problem.
One question: have you set the PORT and SECURE_PORT parameters to the correct values that must match the one used from the browser?

**Comment by alphonse92** [comment]
Of course. I tested it using the browse ip:host/OK

**Comment by alphonse92** [comment]
I have been using moodle 3.4 and the last version of VPL.

**Comment by jcrodriguez-dis** [comment]
I tried to reproduce the issue using VPL 3.3.5 and get not succeed. I think that the version of Moodle does not influence the problem.
The problem may be related to a network configuration not compatible with VPL, are you using a proxy or NAT with port forwarding to reach the jail server?

**Comment by alphonse92** [comment]
Not really, the jail is in another machine, to avoid the complexity to configure DNS and networking stuff like i described here: https://github.com/jcrodriguez-dis/vpl-xmlrpc-jail/issues/33

That jail is public

**Comment by alphonse92** [comment]
I "resolved" just confinguring the JAIL to listen at 80 instead of a custom port.

**Comment by jcrodriguez-dis** [comment]
Then, please, let me know how to replicate your installation to reproduce the problem.
